### PR TITLE
OSSMDOC-110 cipher suites

### PR DESCRIPTION
**Pull request by neal-timpe**
https://issues.redhat.com/browse/OSSMDOC-110

### CODE REVIEW

File: modules/ossm-security-cipher.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-security-cipher_{context}"]
+= Configuring cipher suites and ECDH curves
+
+Cipher suites and Elliptic-curve Diffie–Hellman (ECDH curves) can help you secure your service mesh when you are using TLS 1.2 or earlier. You can define a comma separated list of cipher suites using `spec.istio.global.tls.cipherSuites` and ECDH curves using `spec.istio.global.tls.ecdhCurves` in your `ServiceMeshControlPlane` resource. If either of these attributes are empty, then the default values are used. 
+
```
Comment (knrc) [inline modules/ossm-security-cipher.adoc:9]: Perhaps add a note along the lines of "Please note the cipherSuites setting will have no effect when negotiating TLS 1.3"

**Review by knrc — APPROVED** [review]

File: modules/ossm-security-cipher.adoc (hunk 1 above)
Comment (knrc) [reply modules/ossm-security-cipher.adoc:9]: @neal-timpe just a suggestion

**Review by JStickler — APPROVED** [review]
LGTM

**Comment by tvieira** [comment]
@neal-timpe can I ask a cosmetic change to go with this PR?

```minProtocolVersion``` rather than ```minProtocolversion```

on https://ossm1152udate--ocpdocs.netlify.app/openshift-enterprise/latest/service_mesh/service_mesh_user_guide/ossm-security.html#ossm-security-min-max-tls_ossm-security

**Comment by neal-timpe** [comment]
@tvieira done.

**Comment by tvieira** [comment]
@neal-timpe one more thing in the same file I mentioned above:

The last sentence before the TLS value tables, does not sound right to me:

```The valid values correspond to TLS versions. The default is auto and does not specify a version of TLS.```

Also, the value ```auto``` in the sentence shouldn't be changed to ```TLS_AUTO```?

**Comment by neal-timpe** [comment]
@tvieira You're right. I've made that change.

**Review by tvieira — APPROVED** [review]
lgtm

**Comment by yxun** [comment]
should we add more example lines in modules/ossm-security-mtls.adoc like the following ?
```
apiVersion: maistra.io/v1
kind: ServiceMeshControlPlane
spec:
  istio:
    global:
      tls:
        minProtocolVersion: TLSv1_0
        maxProtocolVersion: TLSv1_2
        ecdhCurves: ...
        cipherSuites: ...

```

**Review by gbaufake — APPROVED** [review]

**Pull request merged**